R/score.R

```diff
@@ -110,53 +97,114 @@
 #' PLoS Comput Biol 15(2): e1006785. \doi{10.1371/journal.pcbi.1006785}
 #' @export
 
-score <- function(data,
-                  metrics = NULL,
-                  ...) {
+score <- function(data, ...) {
```

Comment:
why is metrics not an arg for score?

Reply:
I wasn't quite sure what the default should be. No default (but then again the default is in the methods)? `NULL`? I was hoping it would be more clear that way when looking at the documentation for the methods.